lib/ReactViews/Analytics/PolygonParameterEditor.jsx

```diff
@@ -38,7 +38,7 @@ const PolygonParameterEditor = React.createClass({
     },
 
     getValue() {
-        const pointsLongLats = this.props.parameterValues[this.props.parameter.id];
+        const pointsLongLats = this.props.parameterValues[this.props.parameter.id][0];
```

Comment:
I found a `defaultValue` in `PolygonParameter`. There it's listed in the doc string as:
```@param {Boolean} [options.defaultValue] The default value.```
Is that right? I suspect I've made a mistake in my code tracing somewhere - doesn't your new code expect an array whose first element is `pointsLongLats`?